Indexing/CodeTree.cpp

```diff
@@ -836,7 +836,7 @@ void CodeTree::compileTerm(Term* trm, CodeStack& code, CompileContext& cctx, boo
 	  sti.right();
 	}
 	else {
-	  code.push(CodeOp::getTermOp(CHECK_FUN, s.term()->functor()));
```

Comment:
Although I'm not sure whether CodeTree is currently in use in normal operation, we should check these changes. What led to these suggestions?

Reply:
Hang on, I'll comment each file! This one is because `t` is set to `s.term()` (pure getter) earlier, but never used.

Reply:
@selig I think only forward demodulation currently uses code trees (the `DEMODULATION_LHS_SUBST_TREE` index)